drivers/google/google.go

```diff
@@ -76,6 +77,11 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Value:  defaultUser,
 			EnvVar: "GOOGLE_USERNAME",
 		},
+		mcnflag.StringFlag{
+			Name:   "google-auth-encoded-json",
+			Usage:  "Base64 encoded GCE auth json",
+			EnvVar: "GOOGLE_APPLICATION_CREDENTIALS_ENCODED_JSON",
+		},
```

Comment:
This is net-new correct ? And it just copies your GKE oauth fix.

Reply:
Yes, that's correct.